client/src/inside/components/mar/med-order.js

```diff
@@ -26,6 +26,7 @@ class ScheduleOptions {
     let value = this.OPTIONS[key]
     if(!value) {
       console.error('Could not find a medication schedule for key', key)
+      throw new Error(`Could not find a medication schedule for key: ${key}`)
```

Comment:
@viniciusarre now or during your planned refactoring can you add a test for this throw. Plus, let's make sure the caller is able to handle this throw too.

Reply:
This was added previously... please, refer to line 61 ('Invalid scheduling properly throws' test)